### ISSUE TITLE
aiohttp.web: error: module 'C' not found using command line interface on Windows

### ISSUE THREAD

**Issue by mr-rodgers** [aio-libs/aiohttp#837]
## Long story short

I get `aiohttp.web: error: module 'C' not found` when attempting to use the command line interface, no matter what my arguments are.

Issue seems to be caused by `python -m` passing the location of the web.py script as the first item to sys.argv, and the parsing code doesn't seem to account for that.
## Expected behaviour

`python -m aiohttp.web my.module:func` should _at least_ try to find `my.module`. 
## Actual behaviour

The script tries to find a module named `C` instead, and errors:

```
> python -m aiohttp.web "glotpod.api:Application"
usage: aiohttp.web [-h] [-H HOSTNAME] [-P PORT] entry-func
aiohttp.web: error: module 'C' not found
```

Note that I get the same error too when I don't pass any arguments, and I shouldn't expect that either:

```
> python -m aiohttp.web
usage: aiohttp.web [-h] [-H HOSTNAME] [-P PORT] entry-func
aiohttp.web: error: module 'C' not found
```
## Your environment

Windows 10 64-bit, 32-bit Python, run from inside a pyvenv virtual environment. Using aiohttp 0.21.4.

**Comment by asvetlov** [comment]
Perhaps you have something like `import C` or `from C import *` in your code.
`aiohttp` itself doesn't contain `C` module at all.

**Comment by mr-rodgers** [comment]
This isn't the case. Notice that in the second example, I don't give it an argument at all to locate my code, and it still gives this exact error. 

As indicated, I suspect that this is coming from the first item that Python puts in sys.argv; it's the location on the system of the script that's being run. The location of the aiohttp library on my system is indeed on my `C:` drive, so if the script takes the first item in sys.argv and tries to use it as my init-func (mod:func) argument, it could conceivably always see C as my module argument, explaining this behavior.

**Comment by jashandeep-sohi** [comment]
Normally, the first arg is the script name. We do account for that in `aiohttp.web.py`. Can you print the output of `python -m test` for the following `test.py` file:

```
if __name__ == "__main__":
    import sys
    print(sys.argv)
```

**Comment by mr-rodgers** [comment]
I've identified the problem. The version of web.py installed on my system (from PyPI) uses `sys.argv` and not `sys.argv[1:]` as in the master branch. That change seems fairly recent, so I'm guessing it hasn't made it to PyPi yet. I'm closing this.

**Comment by jashandeep-sohi** [comment]
There have been a couple of minor bug-fix releases since then, but you'd have to ask @asvetlov why this fix wasn't included in them.

**Comment by asvetlov** [comment]
I did not cherry-pick the fix into 0.21.
Have published 0.21.5 release for this.

**Comment by mr-rodgers** [comment]
Hey thanks! That is very convenient.
